apps/studio/src/pages/dashboard/index.tsx

```diff
@@ -1,17 +1,281 @@
+        border="1px"
+        borderColor="base.divider.medium"
+        bgColor="#FFF"
+      >
+        <Table>
+          <Thead>
+            <Tr>
+              <Th w="5%">
+                {/* checkbox */}
+                <Checkbox size="sm" w="fit-content" h="fit-content" />
+              </Th>
+              <Th textTransform="none" w="65%">
+                <HStack>
+                  <Text textStyle="body-2">Title</Text>
+                </HStack>
+              </Th>
+              <Th textTransform="none" w="8%">
+                <HStack>
+                  <Text textStyle="body-2">Status</Text>
+                </HStack>
+              </Th>
+              <Th textTransform="none" w="22%">
+                <HStack>
+                  <Text textStyle="body-2">Last Edited</Text>
+                </HStack>
+              </Th>
+            </Tr>
+          </Thead>
+          <Tbody>
+            {dataToDisplay
+              .slice(
+                (pageNumber - 1) * entriesPerPage,
+                pageNumber * entriesPerPage,
+              )
+              .map((element) => {
+                return (
+                  <Tr>
+                    <Td w="min-content">
+                      <Checkbox size="sm" w="fit-content" h="fit-content" />
+                    </Td>
 
-      <Button onClick={onOpen}>Create new page</Button>
-      <PageCreateModal isOpen={isOpen} onClose={onClose} />
-    </div>
+                    <Td>
+                      <HStack spacing="0.75rem">
+                        {element.type === 'page' &&
+                          element.permalink === '/' && (
+                            <BiHome size="1.25rem" />
+                          )}
+                        {element.type === 'page' &&
+                          element.permalink !== '/' && (
+                            <BiFileBlank size="1.25rem" />
+                          )}
+                        {element.type === 'folder' && (
+                          <BiFolder size="1.25rem" />
+                        )}
+
+                        <VStack alignItems="flex-start">
+                          <Text textStyle="subhead-2">{element.name}</Text>
+                          <Text textStyle="caption-2">
+                            {element.type === 'page' && element.permalink}
+                            {element.type === 'folder' && '0 pages'}
+                          </Text>
+                        </VStack>
+                      </HStack>
+                    </Td>
+
+                    <Td>
+                      {/* Fill doesn't seem to work with semantic tokens   */}
+                      {element.type === 'page' && element.status == 'draft' && (
+                        <Badge
+                          variant="subtle"
+                          colorScheme="warning"
+                          borderRadius="50px"
+                        >
+                          <BiSolidCircle
+                            fill="#FFDA68"
```

Comment:
why do we have a fill for the draft circle but not the published one?

Reply:
The default color scheme of the badge applies the correct fill. Will put in the fill for the published circle for completeness sake!

Reply:
I have updated to wrap it in Chakra's icon component. Semantic tags now work and we no longer need the inline style tag.